src/pages/JoinStep2/JoinStep2.scss

```
@@ -0,0 +1,199 @@
+                                background-color: white;
+                                border: 1px black solid;
+                                border-radius: 3px;
+                                margin-right: 3px;
+                                font-weight: 600;
+                            }
+                        }
+                        .emailInputWrapper {
+                            width: 100%;
+                            @include flex-spaceBetween-center;
+                            input {
+                                width: 100%;
+                                height: 40px;
+                                padding: 0 20px;
+                                line-height: 38px;
+                                border: 1px #ccc solid;
+                                border-radius: 1px;        
+                                font-size: 14px;
+                                margin-left: -8px;
+                            }
+                            .idError {
+                                font-size: 8px;
+                                color: red;
+                                margin-left: -8px;
+                            }
+                            .idValid {
+                                display: none;
+                            }
+                            button {
+                                width: 136px;
+                                height: 38px;
+                                background-color: white;
+                                border: 1px black solid;
+                                border-radius: 3px;
+                                margin-right: 3px;
+                                font-weight: 600;
+                            }
+                        }
+                        .txtAgreement {
+                            width:100%;
+                            height: 107.4px;
+                            margin: 20px 0 0 -17px;
+                            
+                            span {
+                                height: 16px;
+                                display: flex;
+                                align-items: center;
+                                margin-bottom: 5px;
+                                
+                                input {
+                                    width: 16px;
+                                    height: 16px;
+                                }
+                                label {
+                                    margin-left: 5px;
+                                    font-size: 14px;
+                                }
+                            }
+                            p {
+                                width: 450px;
+                                display: flex;
+                                color: #999999;
+                                font-size:13px;
+                                margin-top: 15px;
+                                flex-wrap: wrap;
+                            }
```

```suggestion
                            p {
                                width: 450px;
                                display: flex;
                                flex-wrap: wrap;
                                margin-top: 15px;
                                color: #999999;
                                font-size:13px;
                            }
```
레이아웃에 영향을 주는 속성일 수록 위로 올려주시고, 유사한 속성끼리는 붙여주세요